addon/data/selector-ui.js

```diff
@@ -159,6 +159,72 @@ const ui = (function () { // eslint-disable-line no-unused-vars
     boxBottomEl: null
   };
 
+  /** Displays the box which allows the user to choose between selecting a region,
+      archiving the page, or going to my shots.
+  */
+  exports.SelectMode = {
+    display: function (callbacks) {
+      if (document.querySelector(".pageshot-preview-overlay") !== null) {
+        return;
+      }
+      let oldNode = document.getElementById("pageshot-select-mode-background");
+      if (oldNode !== null) {
+        oldNode.parentNode.removeChild(oldNode);
+      }
+
+      if (this.selectModeBackground === undefined) {
+        this.selectModeBackground = document.createElement("div");
+        this.selectModeBackground.id = "pageshot-select-mode-background";
+        this.selectModeBackground.className = "pageshot-select-mode-background";
+        this.selectMode = document.createElement("div");
+        this.selectMode.className = "pageshot-select-mode";
```

Comment:
Yes, they are all children. They should probably all be local variables instead.